documentation/source/includes/partials_schemas/_response_phone_channels_list.md

```diff
@@ -183,27 +187,27 @@
   "additionalInfo": "string",
   "phones": [
     {
-      "countryCode": "string",
+      "countryCallingCode": "string",
       "areaCode": "string",
       "number": "string",
       "additionalInfo": "string"
     }
-  ],
+  ]
 }
 ```
 
 |     Nome        |  Tipo                                                   | Obrigatório |                            Definição                                                  | Restrições                           |
 |:------------    |:---------------------------------                       |:----------- |:--------------------------------------------------                                    |:------------------------------------ |
 | type            | [Enum PhoneChannelsType](#schemaPhoneChannelsType)      | Sim         | Tipo de canal telefônico de atendimento.                                              |  O Tipo de Canal determina o Tipo de Acesso a ele relacionado:  telefone da central, telefone do SAC, telefone da ouvidoria. |
-| additionalInfo  | string                                                  | Não         | Campo de texto livre para descrever quando o tipo de canal de atendimento for Outros  | Só será preenchido quando o tipo de canal de atendimento for Outros |
+| additionalInfo  | string                                                  | Não         | Texto livre para complementar informação relativa ao Serviço disponível, quando for selecionada a opção p preenchida a opção 'OUTROS_PRODUTOS_SERVICOS'  | Só será preenchido quando o tipo de canal de atendimento for Outros |
```

Comment:
O seguinte texto tem erros de digitação e redundância.

`for selecionada a opção p preenchida a opção 'OUTROS_PRODUTOS_SERVICOS'`

Reply:
Será corrigido após ver qual é o texto correto com a Marili, porque está dessa forma no dicionário.